bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/security.xml

```diff
@@ -71,7 +90,41 @@
 		</channels>
 	</channel-group-type>
 
-	<channel-group-type id="person-event">
+	<channel-group-type id="sub-event-doorbell">
+		<label>Sub Event Informations</label>
+		<channels>
+			<channel id="type" typeId="event-type"/>
+			<channel id="time" typeId="timestamp">
+				<label>Sub-Event Timestamp</label>
+				<description>Moment when then sub-event occurred.</description>
+			</channel>
+			<channel id="message" typeId="message"/>
+			<channel id="snapshot-url" typeId="event-picture-url"/>
+			<channel id="vignette-url" typeId="event-picture-url"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="last-event-doorbell">
+		<label>Last Event</label>
```

Comment:
For consistency with sub event: Should it be "Last Event Information", or should other one be just "Sub Event"? Perhaps adding a description could eliminate the need for "Information"?

Reply:
IIRW descriptions does not appear in the UI.